### PR TITLE
Add Disney profile & logo

### PR DESCRIPTION
**Pull request by ekumlin**

### CODE REVIEW

File: public/employers/disney.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+aliases:
+  - Walt Disney Productions
+  - Walt Disney Studio
+shortName: Disney
+ticker: DIS
+wiki: The Walt Disney Company
+image: disney.svg
+officialWebsite: https://thewaltdisneycompany.com/about/
+industries:
+  - entertainment
```
Comment (frosteeze) [inline public/employers/disney.yml:11]: Their main industry is entertainment, but they are also in hospitality/lodging, media, and some retail like apparel or clothing.

File: public/employers/disney.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    positivity: -1
+    type: publication
+    sources:
+      - source: San Gabriel Valley Tribune
+        link: https://web.archive.org/web/20200421210239/https://www.sgvtribune.com/2020/04/20/disney-to-stop-paying-half-its-workforce-but-is-still-on-track-to-deliver-1-5-billion-dividend/
+        date: 2020-04-20T18:58:44Z
+      - source: Los Angeles Times
+        link: https://www.latimes.com/business/story/2020-04-19/disney-stops-paying-100000-workers
+        date: 2020-04-19T16:15:17Z
+  - summary: Despite resorting to furloughs, Disney intends to continue paying executive bonuses and a $1.5-billion-USD stock dividend this year.
```
Comment (frosteeze) [inline public/employers/disney.yml:42]: DIS is in many retirement ETFs and they need to give out dividends as to keep these ETFs stable iirc, just so you know.

Comment (ekumlin) [reply public/employers/disney.yml:42]: This is true. I'd stand by this as a `-1` though for two reasons: that they're also paying out bonuses, and that retirement ETFs still have time to recover, while workers relying on those paychecks *now* aren't able to get the money they need.

File: public/employers/disney.yml (hunk 1 above)
Comment (ekumlin) [reply public/employers/disney.yml:11]: Yeah this is a tough call. Wikipedia only puts them as media and entertainment. But I think apparel and hospitality would be two others to include as well.

**Pull request merged**